src/status_im/common/controlled_input/utils.cljs

```diff
@@ -15,6 +15,10 @@
   (:value state))
 
 (defn numeric-value
+  [state]
+  (or (parse-double (input-value state)) 0))
+
+(defn value-bn
```

Comment:
Just for the sake of similarity I'd name this as `bn-value`, wdyt?

Reply:
Yeah, that's correct. I will clean this up 🙏